src/test/java/org/openrewrite/java/migrate/apache/commons/lang/ApacheCommonsStringUtilsTest.java

```diff
@@ -0,0 +1,266 @@
+                      string = StringUtils.mid(in, 3, 4);
+                      string = StringUtils.overlay(in, "overlay", 3, 5);
+                      string = StringUtils.remove(in, "r");
+                      string = StringUtils.repeat(in, 4);
+                      string = StringUtils.repeat(in, ",", 4);
+                      string = in == null ? null : in.replaceFirst(Pattern.quote("search"), "replacement");
+                      string = in == null ? null : new StringBuilder(in).reverse().toString();
+                      string = StringUtils.right(in, 5);
+                      string = StringUtils.rightPad(in, 5);
+                      string = StringUtils.rightPad(in, 5, ' ');
+                      string = StringUtils.rightPad(in, 5, " ");
+
+                      array = in == null ? null : in.split(", ");
+                      bool = StringUtils.startsWith(in, "prefix");
+                      bool = StringUtils.startsWithAny(in, "prefix");
+                      bool = StringUtils.startsWithIgnoreCase(in, "prefix");
+                      array = StringUtils.stripAll(in);
+
+                      string = in == null ? null : (in.endsWith("suffix") ? in.substring(0, in.lastIndexOf("suffix")) : in);
+                      string = in == null ? null : (in.startsWith("chars") ? in.substring("chars".length()) : in);
+
+                      bool = StringUtils.startsWith(in, "prefix");
+
+                      array = in == null ? null : in.split(" ");
+
+                      string = in == null ? null : in.trim();
+                      string = in == null ? null : in.substring(in.indexOf("|") + 1, in.length());
+                      string = in == null ? null : in.substring(2, 4);
+                      string = in == null ? null : in.chars().map((c) -> Character.isUpperCase(c) ? Character.toLowerCase(c) : Character.toUpperCase(c)).collect(StringBuilder::new, StringBuilder::appendCodePoint, StringBuilder::append).toString();
+                      string = in != null ? in.trim() : "";
+                      string = in == null ? null : (in.trim() == null ? null : in.trim());
+                      string = in == null ? null : in.trim();
+                      string = in == null ? null : in.toUpperCase();
+                  }
+              }
+              """
+          )
+        );
+    }
+
+    @Test
+    void canCallMethodOnResult() {
+        rewriteRun(
+          //language=java
+          java(
+            """
+              import org.apache.commons.lang3.StringUtils;
+                            
+              class Foo {
+                  void test(String s) {
+                      String test = StringUtils.strip(s).toString();
+                  }
+              }
+              """,
+            """
+              class Foo {
+                  void test(String s) {
+                      String test = (s == null ? null : in.trim()).toString();
```

Comment:
@knutwannheden this test currently fails; it's similar to a case we've discussed before on whether the `JavaTemplate`
 should itself detect whether a statements needs to be wrapped in braces. In this synthetic example we're changing behavior: before it would have thrown a NPE when the input is `null`, after non-braced replacement it no longer would. We should not make such behavioral changes.
```suggestion
                      String test = s == null ? null : in.trim().toString();
```
https://ge.openrewrite.org/s/jwi2f75wlzjts/tests/task/:test/details/org.openrewrite.java.migrate.apache.commons.lang.ApacheCommonsStringUtilsTest/canCallMethodOnResult()?top-execution=1